File: src/main/java/xyz/oribuin/lilori/commands/author/CmdShutdown.java
package xyz.oribuin.lilori.commands.author;

import xyz.oribuin.lilori.managers.command.Command;
import xyz.oribuin.lilori.managers.command.CommandEvent;

import java.util.Collections;

public class CmdShutdown extends Command {

    public CmdShutdown() {
        this.name = "Shutdown";
        this.description = "Shutdown the bot.";
        this.aliases = Collections.emptyList();
        //this.arguments = "[None]";
        this.ownerOnly = true;
    }

    @Override
    public void executeCommand(CommandEvent event) {
        String[] args = event.getMessage().getContentRaw().split(" ");

        event.getChannel().sendMessage("**Shutting down bot**").queue();
        event.getJDA().shutdown();


    }
}

File: src/main/kotlin/xyz/oribuin/lilori/commands/author/CmdShutdown.kt
package xyz.oribuin.lilori.commands.author

import xyz.oribuin.lilori.handler.Command
import xyz.oribuin.lilori.handler.CommandEvent

class CmdShutdown : Command() {
    init {
        name = "Shutdown"
        description = "Shutdown the bot."
        aliases = emptyList()
        //this.arguments = "[None]";
        isOwnerOnly = true
    }

    override fun executeCommand(event: CommandEvent?) {
        (event ?: return)

        event.channel.sendMessage("**Shutting down bot**").queue()
        event.jda.shutdown()
    }
}